src/lib/core/fiber.h

```diff
@@ -584,8 +574,20 @@ struct fiber {
 	struct fiber_slice max_slice;
 	/** Valgrind stack id. */
 	unsigned int stack_id;
-	/* A garbage-collected memory pool. */
+	/** A garbage-collected memory pool. */
 	struct region gc;
+#ifdef ENABLE_BACKTRACE
+	/**
+	 * Backtrace of the first fiber gc allocation that found
+	 * leaked on inspection.
+	 */
+	struct backtrace *leak_bt;
```

Comment:
It's not necessary the backtrace of a leak. It's the backtrace of the first allocation. Let's name it accordingly.

Reply:
Renamed to `first_alloc_bt`.